### DISCUSSION TITLE
DATGEN DR12 SETUP

### DISCUSSION THREAD

**Discussion by prophetcoffee** [artisan-roaster-scope/artisan#1637] [Q&A]
TLDR: Trying to setup digital control from Artisan for a second hand Datgen DR12. Cannot operate the machine without Artisan control.

>>>>>>>>>>>

New to Artisan community after getting a secondhand Datgen roaster. Functionally speaking, it's in great condition. Was moderately used for a couple of years, and has since been kept in an air sealed shipping container. It even had a few great spec upgrades (air pressure gauge, extra thermocouple installed, variable fan speed). 

Unfortunately, there's little to no information online about these machines; and from Reddit threads it seems like maybe the company isn't in operation anymore? Datgens own website shares very little about these machines. 

I've hooked up gas, duct work, taken the machine apart and deep cleaned, and now I can't figure out how to turn it on - as the early models don't seem to have manual operation capabilities. Anyone know about these, or what software or plug-in I'm missing through Artisan to access Datgen controls?

**Comment by poundy** [comment]
Unlesss someone else has one, about the limits here are https://artisan-scope.org/machines/datgen/
From that, you need a USB-Serial adapter that connects to the device's MODBUS network and should give you BT and ET. To me that should completely mean the roaster is manually controlled and no control interaction occurs with Artisan. https://www.youtube.com/watch?v=MSn2aNS_0uk seems relevant

**Reply by prophetcoffee** [reply]
It seems that in their initial release, Datgen DR12s had a function through Artisan to fully control the variable speed components of our roaster through standard MODBUS communication. Whether this was setup by Artisan, or manually through the guidance of the manufacturer, I am uncertain. I am certain however, that this function does exist, as we've found past DR12s Artisan profiles deep in the depths of the internet. We were able to download the profiles and gain access to the machine control, and the variable speed functions. Our machine turned right on, as did our pilot flame and drum fan. Now we are working to fine tune these controls, as the settings were not perfect, and we also never synced up the gas release function...but we are making progress nonetheless!

**Reply by prophetcoffee** [reply]
The early models of these machine only have half of the machine controls shown in this video! The only thing you can manually turn on without artisan is the cooling bin fan and agitator - the other controls simply run power to the components but to not start them.

**Comment by peterweiser** [comment]
Hi ProphetCoffee,

I ve exactly the same maschine, same story and had same questions.
I ve also bought an used early Dätgen DR12 in quite a good condition and didn´t even know how to turn it on.
Unfortunately - it´s hard to reach the owner as he´s not really chatty. I also don´t know if the company is still in business or not. He was the only exhibitor at the World Of Coffee in Milano 2 years ago, which had a booth for the whole weekend, but didn´t even come.
But nevertheless - the maschine is great and I m able to produce high quality roasts with it. I m running this business successfully since 2020 and our customers love the coffee.
If you have any question, and if your maschine is similar to mine (Mr. Dätgen once told me about many technical changes, so I think there are different versions of the DR12 out), maybe I m able to help you - also with the artisan configuration. Dont hesitate to contact me.
Peter

**Reply by prophetcoffee** [reply]
Hey Peter,

Thanks for the reply! In our weeks of searching the internet, reaching out to Datgen users past and present, and talking with both Artisan and many other coffee roasting consultants...you are the first person we've found that has the early Datgen DR12 with no built in machine controls! We are in rare company, it seems. Could you please email me at zach@prophet-coffee.com ? Would love to hear about your experience and specifically how you configured the controls through artisan. We have made some progress since posting!
